.github/workflows/tests.yml

```diff
@@ -0,0 +1,22 @@
+# This ensures our end to end tests run for production builds
+name: E2E Tests
```

Comment:
Love that tests are being run against PRs now in our pipeline 😁

Reply:
Same same! Really stoked about this! I didn't set up the config for unit/component testing because I figured we'd make use of the e2e tests for now. But we can always jump back into this later if we want to add it. I'll add a quick note in the readme for creating and running tests 😁